### PR TITLE
feat: update alert component

### PR DESCRIPTION
**Pull request by stevenpersia**
https://github.com/user-attachments/assets/93f5578e-4a82-4a52-b516-197c3cb26617

### CODE REVIEW

File: packages/Alert/src/index.tsx [hunk 1]
```diff
@@ -44,13 +51,41 @@ const AlertComponent = forwardRef<'div', AlertProps>(
...
+    const layout: { bottom: BoxProps; right: BoxProps } = {
+      bottom: {
+        alignItems: 'flex-start',
+        flexDirection: 'column',
+      },
+      right: {
+        alignItems: { _: 'flex-start', md: 'center' },
+        flexDirection: { _: 'column', md: 'row' },
+      },
+    }
```
Comment (theo-mesnil) [inline packages/Alert/src/index.tsx:87]: Can we move to top before the component ?

File: packages/Alert/src/index.tsx [hunk 2]
```diff
@@ -74,34 +109,36 @@ const AlertComponent = forwardRef<'div', AlertProps>(
...
-              <div>{content}</div>
-              {cta}
-            </Box>
-          ) : (
-            content
-          )}
+        <Box flex={1}>
+          <Box display="flex" gap="md" justifyContent="space-between" {...layout[ctaPosition]}>
+            <Box flex={1}>{content}</Box>
+
```
Comment (theo-mesnil) [inline packages/Alert/src/index.tsx:118]: remove empty line

File: packages/Alert/src/index.tsx [hunk 3]
```diff
@@ -74,34 +109,36 @@ const AlertComponent = forwardRef<'div', AlertProps>(
...
+const AlertSecondaryAction = forwardRef<'button', Omit<ButtonProps, 'size' | 'variant'>>(
+  (props, ref) => <Button flexShrink={0} ref={ref} w="fit-content" {...props} variant="tertiary" />
+)
 
-export const Alert = Object.assign(AlertComponent, { Title, Button: AlertButton })
+export const Alert = Object.assign(AlertComponent, {
+  Title,
+  PrimaryAction: AlertPrimaryAction,
+  SecondaryAction: AlertSecondaryAction,
```
Comment (theo-mesnil) [inline packages/Alert/src/index.tsx:143]: I think this is a breaking change, maybe we can for now export `Button: AlertPrimaryAction` ? With a deprecated message ?

Comment (theo-mesnil) [reply packages/Alert/src/index.tsx:143]: Or rename` Button / SecondaryButton` ?

File: packages/Alert/src/index.tsx (hunk 1 above)
Comment (stevenpersia) [reply packages/Alert/src/index.tsx]: You're right 👍 Fixed here : b3e2450.

File: packages/Alert/src/index.tsx (hunk 2 above)
Comment (stevenpersia) [reply packages/Alert/src/index.tsx]: Done here b3e2450.

File: packages/Alert/src/index.tsx (hunk 3 above)
Comment (stevenpersia) [reply packages/Alert/src/index.tsx]: Nice catch! Yes, I tried to reduce the breaking changes on this component, but I forgot about that. You are absolutely right. I kept the current button and put the second one (see b3e2450). So, maybe we don't need deprecated anymore, it's just a styling update?

**Review by theo-mesnil — APPROVED** [review]

**Review by theo-mesnil — APPROVED** [review]

**Pull request merged**